testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -47,6 +47,20 @@ Feature: Smoke tests for <client>
     Then I should see a "1 patch update has been scheduled for" text
     And I wait until event "Patch Update:" is completed
 
+@slemicro53_minion
+@slemicro53_ssh_minion
+@slemicro52_minion
+@slemicro52_ssh_minion
+  Scenario: Reboot the <client> and wait until reboot is completed
+    Given I am on the Systems overview page of this "<client>"
+    When I follow first "Schedule System Reboot"
+    Then I should see a "System Reboot Confirmation" text
+    And I should see a "Reboot system" button
+    When I click on "Reboot system"
+    Then I should see a "Reboot scheduled for system" text
+    And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
+    Then I should see a "This action's status is: Completed" text
```

Comment:
For such special repetitive scenario, I would wrap all the steps in one step.

Reply:
Ok, I will put that into a own step.